here-naksha-lib-core/src/main/java/com/here/naksha/lib/core/models/naksha/EventHandler.java

```diff
@@ -123,9 +123,15 @@ public void setExtensionId(@Nullable String extensionId) {
   @AvailableSince(NakshaVersion.v2_0_7)
   public @NotNull IEventHandler newInstance(@NotNull INaksha naksha, @NotNull EventTarget<?> eventTarget) {
```

Comment:
I see another "newInstance()" method is not having this extension specific code. So, remove other "newInstance" method and so, we need to remove `EventPipeline.addEventHandler(@NotNull EventHandler eventHandler)` method as well that is using this, else code won't compile

Reply:
fixed as discussed